### PR TITLE
Add form field to stories

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

I've added the form field to the template and each story so users can download the PDF version.

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_includes/layouts/story.njk [hunk 1]
```diff
@@ -65,6 +64,14 @@ layout: layouts/base.njk
                             </ul>
                         </div>
                     </div>
+                    {% if hubspot.formId %}
+                    <div class="flex flex-col mt-6 px-6">
+                        <div class="pt-3 pb-3 flex flex-col">
+                            <h3 class="mb-3">Download webinar slides</h3>
```
Comment (Yndira-FlowForge) [inline src/_includes/layouts/story.njk:70]:
```suggestion
                            <h3 class="mb-3">Download customer story</h3>
```

**Comment by Yndira-FlowForge** [comment]
@Hasmin-AC, I initially planned to address [this](https://flowforgeworkspace.slack.com/archives/C02TG4WH5PC/p1704701959127949?thread_ts=1704255341.202729&cid=C02TG4WH5PC) through a separate PR. However, I realized its connection to this one. Perhaps we can collaborate on creating instructions for adding a new customer story, including guidance on creating the form in HubSpot? I'll need your help with that part.

File: src/customer-stories/leveraging-node-red-and-flowfuse-to-revolutionize-irrigation.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: Leveraging Node-RED and FlowFuse to Revolutionize Irrigation
 image: /images/stories/pidd-view.png
 date: 2023-12-29
 logo:
+hubspot:
+    formId: 1b361db7-cf8e-4574-981b-9de01de04fd9
```
Comment (Hasmin-AC) [inline src/customer-stories/leveraging-node-red-and-flowfuse-to-revolutionize-irrigation.md:7]:
```suggestion
    formId: f77ee5dd-6e7a-4475-aa66-4f1824aab8d7
```

File: src/customer-stories/opto22-embraces-node-red.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ image: /images/stories/opto22.jpg
 date: 2023-06-01
 # This "logo" is for the main page
 logo: 
+hubspot:
+    formId:
```
Comment (Hasmin-AC) [inline src/customer-stories/opto22-embraces-node-red.md:8]:
```suggestion
    formId: ef2e9c57-df05-456e-a612-0531c3241002
```

File: src/customer-stories/stfi-future-of-textile-powered-by-node-red.md [hunk 4]
```diff
@@ -3,6 +3,8 @@ title: The Future of Textile Manufacturing Powered with Node-RED
 image: /images/stories/stfi-future-textile.jpg
 date: 2023-11-14
 logo:
+hubspot:
+    formId:
```
Comment (Hasmin-AC) [inline src/customer-stories/stfi-future-of-textile-powered-by-node-red.md:7]:
```suggestion
    formId: 04a5843c-1d39-4410-8727-af15b42108d1
```

File: src/customer-stories/un-wmo-nr-data-sharing.md [hunk 5]
```diff
@@ -3,6 +3,8 @@ title: World Meteorological Organization (WMO) Uses Node-RED to Modernize Sharin
 image: /images/stories/un-wmo.jpg
 date: 2023-09-07
 logo: /images/stories/logos/logo_wmo.png
+hubspot:
+    formId:
```
Comment (Hasmin-AC) [inline src/customer-stories/un-wmo-nr-data-sharing.md:7]:
```suggestion
    formId: 37f86f46-62d1-4203-bce9-0318ce993710
```

**Review by Hasmin-AC — CHANGES_REQUESTED** [review]
updated the form IDs

File: src/_includes/layouts/story.njk [hunk 6]
```diff
@@ -65,6 +64,14 @@ layout: layouts/base.njk
                             </ul>
                         </div>
                     </div>
+                    {% if hubspot.formId %}
+                    <div class="flex flex-col mt-6 px-6">
+                        <div class="pt-3 pb-3 flex flex-col">
+                            <h3 class="mb-3">Download customer story</h3>
```
Comment (Yndira-FlowForge) [inline src/_includes/layouts/story.njk:70]:
```suggestion
                            <h3 class="mb-3">Download Case Study</h3>
```

**Review by ZJvandeWeg — APPROVED** [review]

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**